src/objective-c/BoringSSL-GRPC.podspec

```diff
@@ -0,0 +1,4518 @@
+# BoringSSL CocoaPods podspec
```

Comment:
Looks like this file is new in this PR and it looks like it's autogenerated, but I don't see the corresponding .template file. Is that intended?

Reply:
Intended but I didn't like it too. I just got an idea to make it .template and updated the PR. PTAL.